runtime/lua/vim/lsp/completion.lua

```diff
@@ -0,0 +1,776 @@
+local M = {}
```

Comment:
@lewis6991 How can I make the public stuff from this module appear in `lsp.txt`?

Reply:
would need an entry here: https://github.com/neovim/neovim/blob/32085c1e7c4936097c76970fa3639661b66c7e78/scripts/gen_vimdoc.lua#L268